lms/templates/courseware/progress.html

```diff
@@ -29,181 +29,174 @@
+                    %for chapter in courseware_summary:
+                        %if not chapter['display_name'] == "hidden":
+                        <div>
+                            <h4 class="hd hd-4" id="chapter_${loop.index}">${ chapter['display_name'] | h}</h4>
+                            <div class="sections">
+                                %for section in chapter['sections']:
+                                    <div>
+                                        <%
+                                        earned = section.all_total.earned
+                                        total = section.all_total.possible
+                                        percentageString = "{0:.0%}".format( float(earned)/total) if earned > 0 and total > 0 else ""
+                                        %>
+                                        <h5 class="hd hd-5">
+                                            <a href="${reverse('courseware_section', kwargs=dict(course_id=course.id.to_deprecated_string(), chapter=chapter['url_name'], section=section.url_name)) | h}">
+                                                ${ section.display_name | h}
+                                                %if total > 0 or earned > 0:
+                                                <span class="sr">
+                                                    ${_("{earned} of {total} possible points").format(earned='{:.3n}'.format(float(earned)), total='{:.3n}'.format(float(total))) | h}
+                                                </span>
+                                                %endif
+                                            </a>
+                                            %if total > 0 or earned > 0:
+                                            <span> ${"({0:.3n}/{1:.3n}) {2}".format( float(earned), float(total), percentageString ) | h}</span>
+                                            %endif
+                                        </h5>
+                                        %if section.due is not None:
+                                        <p>
+                                            <%
+                                            formatted_string = get_time_display(section.due, course.due_date_display_format, coerce_tz=settings.TIME_ZONE_DISPLAYED_FOR_DEADLINES)
+                                            due_date = '' if len(formatted_string)==0 else _(u'due {date}').format(date=formatted_string)
+                                            %>
+                                            <em>${due_date | h}</em>
+                                        </p>
+                                        %endif
+                                        <div class="scores">
+                                            %if len(section.scores) > 0:
+                                            <h6 class="hd hd-6">${ _("Problem Scores: ") if section.graded else _("Practice Scores: ")}</h6>
```

Comment:
Because we want to avoid non-unique heading values, and I think we are on the verge of over-saturating this page with headings in general, I think we should use a paragraph here instead of an h6.  Ideally, we would use a description list here: https://www.w3.org/TR/html5/grouping-content.html#the-dl-element  Problem Scores being the ```<dt>``` and the list that follows would be the ```<dd>```